src/opm/input/eclipse/Schedule/Network/NetworkKeywordHandlers.cpp

```diff
@@ -170,6 +172,29 @@ void handleNODEPROP(HandlerContext& handlerContext)
         }
 
         if (as_choke) {
+            const auto& group = handlerContext.state().groups.get(name);
```

Comment:
with this line, you should probably delete the line 207 below. 
![image](https://github.com/OPM/opm-common/assets/6847941/b87d7dbf-8569-45e5-b0bd-a4b5bc4fd9df)

Reply:
Deleted the line.